en/docs/learn/design-api/mock-api/create-a-mock-api-with-an-inline-script.md

```diff
@@ -92,14 +92,14 @@ For this let's use the following OpenAPI URL: `https://petstore.swagger.io/v2/sw
       // **MANUALLY ADDED CODE** //
```

Comment:
can you please check the above // **GENERATED CODE** // as well and change if it is wrong.

Reply:
https://github.com/wso2/docs-apim/pull/2162/commits/6ede044b560098a4ab77e0ad85bd1359d70e13f8